test/test_add_products.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+from src.models import ProductCreate
+
+client = TestClient(app)
+
+
+@pytest.fixture
+def product_data():
+    return {
+        'name': 'Product Name',
+        'price': 9.99,
+        'type': 'food',
+        'region': 'Europe'
+    }
```

Comment:
You should place fixtures in `conftest.py` file

Reply:
Actually, you don't need to create fixture for data. Otherwise you'll have A LOT of fixtures, it will be total mess. Just create product request data in the test you run the code